lib/widgets/picos_select.dart

```diff
@@ -107,12 +107,14 @@ class _PicosSelectState extends State<PicosSelect> {
         absorbing: widget.disabled,
         child: ButtonTheme(
           alignedDropdown: true,
+
```

Comment:
Why is an empty line here?

Reply:
This does not belong to my changed files, but no problem I can change that ;)